### ISSUE TITLE
doc: PDF generation broken

### ISSUE THREAD

**Issue by dbkinder** [zephyrproject-rtos/zephyr#15027]
**Describe the bug**
Generating a PDF version of the Zephyr documentation displays some errors and creates PDF output with bad formatting in the Network Stack Architecture section through the end of that chapter.  

Note that this documentation section uses a floating image to create a two-column effect that renders fine for the HTML docs, but not for the PDF.  I suspect we'll need to add some specific latex/PDF directive guidance as was added for the HTML generation.

![image](https://user-images.githubusercontent.com/11063618/55243985-a99e6900-5216-11e9-90ae-843fdd113367.png)

Here's the output from the ``make pdfdocs``:
```
[5/6] Generating PDF documentation
FAILED: latex/zephyr.pdf
cd /home/dbkinder/zephyr/doc/_build/latex && /home/dbkinder/.local/lib/python3.6/site-packages/cmake/data/bin/cmake -E env LATEXOPTS="-halt-on-error\ -no-shell-escape" /usr/bin/latexmk -quiet -pdf -dvi- -ps-
File::Glob::glob() will disappear in perl 5.30. Use File::Glob::bsd_glob() instead. at /usr/bin/latexmk line 7482.
Latexmk: Run number 1 of rule 'pdflatex'
pdflatex: unrecognized option '-halt-on-error -no-shell-escape'
This is pdfTeX, Version 3.14159265-2.6-1.40.18 (TeX Live 2017/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 256
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs of latex/pdflatex.
ninja: build stopped: subcommand failed.
Makefile:25: recipe for target 'pdfdocs' failed
make: *** [pdfdocs] Error 1
```

**To Reproduce**
Do a ``make pdfdocs``

**Impact**
We currently don't have a requirement to create a PDF version of the documentation, but such a requirement is likely needed in the future.

**Comment by jukkar** [comment]
Even thou this issue is seen with networking documentation, I think it is more about an issue in documentation system itself, so assigning to David for a time being.

**Comment by dbkinder** [comment]
PDF build failure resolved with #19255
